css/properties/visibility.json

```diff
@@ -24,7 +24,7 @@
             "ie": {
               "version_added": "4",
               "notes": [
-                "Internet Explorer doesn't support <code>visibility: initial</code>.",
+                "Internet Explorer doesn't support <code>visibility: initial</code> and <code>visibility: unset</code>.",
```

Comment:
Would you mind inserting this as a new note? I'd make this three total entries in the array, rather than two.

Technically, we ought to have separate subfeatures for each of the named values, but breaking up the note respects that fact a little better in the mean time.